packages/xo-web/src/xo-app/xoa/update/index.js

```diff
@@ -276,6 +327,55 @@ const Updates = decorate([
             </CardBlock>
           </Card>
         </Col>
+        <Col mediumSize={6}>
+          <Card>
+            <CardHeader>{_('releaseChannels')}</CardHeader>
+            <CardBlock>
+              <form id={state.channelsFormId} className='form'>
+                <div className='form-group'>
+                  <Select
+                    autoSelectSingleOption={false} // avoid selecting *unlisted channel* before public channels are retrieved
+                    isLoading={state.channelsOptions === undefined}
+                    onChange={effects.onChannelChange}
+                    options={state.channelsOptions}
+                    placeholder={formatMessage(messages.selectChannel)}
+                    required
+                    value={state.channel}
+                  />
+                  <br />
+                  {state.isUnlistedChannel && (
+                    <div className='form-group'>
+                      <input
+                        autoFocus
+                        className='form-control'
+                        name='unlistedChannel'
+                        onChange={effects.linkState}
+                        placeholder={formatMessage(
+                          messages.unlistedChannelName
+                        )}
+                        required
+                        type='text'
+                        value={helper(
+                          state,
+                          xoaConfiguration,
```

Comment:
Did `xoaConfiguration` contains `unlistedChannel` property?

Reply:
No `xoaConfiguration` contain only the selected channel

Reply:
So, the usage of the helper is not correct.